scanpy/testing/_helpers.py

```diff
@@ -99,7 +101,11 @@ def _prepare_pbmc_testdata(sparsity_func, dtype, small=False):
     small
         False (default) returns full data, True returns small subset of the data."""
 
-    adata = pbmc3k().copy()
+    global pbmc3k
+
+    if pbmc3k is None:
+        pbmc3k = sc.datasets.pbmc3k()
+    adata = pbmc3k.copy()
```

Comment:
I don't understand this change. I also don't think it seems relevant to moving the test utilities.

Could things like this be reverted?

Reply:
That’s the only place where making those “cached dataset” function into fixtures wasn’t beneficial. This helper can be transformed into a fixture in a later PR.

Reply:
Sure, if we go that route. But I’m pretty sure fixtures are the better way.

Reply:
OK, fixturized this too, much better now